### PR TITLE
hayai 0.1.0 (new formula)

### PR DESCRIPTION
**Pull request by scribblemaniac**
hayai is a C++ framework for writing benchmarks for pieces of code in the style of unit testing frameworks such as googletest. It makes it easy to properly benchmark code without cluttering up your main source files with timing functions and loops.

### CODE REVIEW

File: Library/Formula/hayai.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
+class Hayai < Formula
+  desc "A C++ benchmarking framework inspired by the googletest framework"
+  homepage "http://nickbruun.dk/2012/02/07/easy-cpp-benchmarking"
+  url "https://github.com/nickbruun/hayai/archive/v0.1.0.tar.gz"
+  sha256 "739d6dc5126a0b3dfe6c944999f48e7ede823bfdbf1f8801f23af25bf355db65"
+
+  depends_on "cmake" => :build
+  depends_on "autoconf" => :build
```
Comment (DomT4) [inline Library/Formula/hayai.rb]: It looks like the autotools are only needed if we pass `-DTEST=true` to CMake?

Comment (scribblemaniac) [inline Library/Formula/hayai.rb]: That makes sense thanks. Fixed.

**Comment by dunn** [comment]
Very cool, merged as 145d90a600c90dc81cbebeaaa9b81048eb4ef1ac. Thanks!

**Comment by scribblemaniac** [comment]
@dunn Thank you :smile:

**Pull request closed without merging**